core/themes/basis/css/component/small-text-components.css

```diff
@@ -176,22 +176,15 @@ ul.links,
 }
 
 .links.inline li:after {
-  content: ',';
+  content: '•';
   position: static;
   display: inline;
   -webkit-transform: none;
   transform: none;
   border: 0;
+  padding: 0 0 0 0.5em;
```

Comment:
Small nitpick: should't the padding-left be `0.6em`, to be the same value as padding-right for `.links.inline li`?

Reply:
Yes.